loader/stage0/efi/efi_boot_param.c

```diff
@@ -112,6 +115,7 @@ evmm_desc_t *boot_params_parse(tos_startup_info_t *p_startup_info, uint64_t load
 	evmm_desc->sipi_ap_wkup_addr = (uint64_t)p_startup_info->sipi_ap_wkup_addr;
 #ifdef LIB_EFI_SERVICES
 	evmm_desc->system_table_base = (uint64_t)p_startup_info->system_table_addr;
+	evmm_desc->top_of_mem = efi_get_tom();
```

Comment:
I want to keep this efi specific method in efi stage0 instead of put in stage1. In future, we may cleanup stage1 to only boot MP with init-sipi-sipi method.

Reply:
OK